test/oli/institutions_test.exs

```diff
@@ -472,6 +472,37 @@ defmodule Oli.InstitutionsTest do
       assert result_institution.id == first_institution.id
     end
 
+    test "approve_pending_registration_as_new_institution/1 creates a new institution and registration and removes pending registration" do
+      pending_registration =
+        insert(:pending_registration, %{
+          issuer: "new issuer",
+          institution_url: "https://www.something.com"
+        })
+
+      assert Institutions.list_institutions()
```

Comment:
Nit but `assert` succeeds if the argument passed is not `nil`, and `refute` passes if the arg is `nil`.
Therefore, you can avoid the comparison and just do 
```elixir
refute Institutions.list_institutions() |> Enum.find(fn i -> i.institution_url == "https://www.something.com" end)
```

or a few lines below

```elixir
assert Institutions.list_registrations()
             |> Enum.find(fn r -> r.issuer == "new issuer" end)
```

Same in the other tests